packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -1339,6 +1340,67 @@ export function listenToEventResponderEventTypes(
   }
 }
 
+export function createTouchHitTargetElement(
+  bottom: number,
+  left: number,
+  right: number,
+  top: number,
+  rootContainerInstance: Element | Document,
+  parentNamespace: string,
+): HTMLElement {
+  const touchHitTargetElement = ((createElement(
+    'div',
+    EMPTY_OBJECT,
+    rootContainerInstance,
+    parentNamespace,
+  ): any): HTMLElement);
+  const touchHitTargetElementStyle = touchHitTargetElement.style;
+
+  touchHitTargetElementStyle.position = 'absolute';
+  if (top !== 0) {
```

Comment:
We will always need to set each CSS position including for `0`. Here's an [example](https://codepen.io/anon/pen/GLorbm) that will break if `top:0` is removed from the CSS.

I also realised that the effective hitslop is reduced if a border is applied to the container, as the `0` position of the absolutely-position div is determined by the `padding-box`.